cacli/cmd/cancel/cancel.go

```diff
@@ -0,0 +1,53 @@
+package cancel
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/briandowns/spinner"
+	"github.com/cloud-annotations/training/cacli/cmd/login"
+	"github.com/cloud-annotations/training/cacli/e"
+	"github.com/cloud-annotations/training/cacli/ibmcloud"
+	"github.com/spf13/cobra"
+)
+
+func Run(cmd *cobra.Command, args []string) {
+	if len(args) < 1 {
+		e.Exit(errors.New("no `Model ID` provided\nUsage: cacli download <model_id>"))
+	}
+
+	modelID := args[0]
+
+	session := login.AssertLoggedIn()
+
+	model, err := session.GetTrainingRun(modelID)
+	if err != nil {
+		e.Exit(err)
+	}
+
+	switch model.Entity.Status.State {
+	case "completed", "error", "failed", "canceled":
+		// do nothing
+		e.Exit(errors.New("Can not cancel training. use 'cacli list' to check status"))
```

Comment:
errors shouldn't start with capitals + is an error too harsh?  Would probably be okay to silently continue

also if we were to log an error/warning we should just tell them the status since we have it

Reply:
makes sense.